extensions/kubernetes-client/runtime-internal/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientBuildConfig.java

```diff
@@ -161,4 +161,9 @@ public class KubernetesClientBuildConfig {
     @ConfigItem(defaultValue = "true")
     public boolean generateRbac;
 
+    /**
+     * Enable okhttp log messages at STATIC_INIT
+     */
+    @ConfigItem(name = "okhttp.static.logger", defaultValue = "false")
+    public boolean enabled;
```

Comment:
You definitely cannot use something as generic as `enabled` for such a specific configuration. Also the name doesn't really follow the conventions.

Reply:
What name do you suggest?
would be something like, remove the name from annotation and rename the variable to something like "httpLogging"?